README.md

```diff
@@ -1,2 +1,216 @@
+15. If the user was already logged in, the signup or log-in is skipped.
+
+B) In the navbar I can see a button or link to “Become a Host”. If I click this button I will see:
+- The page for signing up (with name, email, and password) or logging in (with email and password).
+- After successfully sing up or logging in, I can see a form to upload all information and photos for a new property (see A.2).
+- If the user was already logged in, the “Become a Host” button takes the user directly to the form, without requesting sign up or log in.
+- After entering the new property info, the page shows the homepage and a message with: “Thank you for uploading your property”.
+- There must be an option for the property owner to block periods of time to avoid users to make reservations on those periods.
+
+C) In the navbar I can see also buttons or links to sign up, log in, and log out.
+
+D) In the navbar I can see a button to show the list of all uploaded properties for the logged-in user. After clicking this button, the page shows the list of property names uploaded by the logged-in user. Each name is a link to the show page of the property. If a name is clicked, the page will show:
+- The name of the property,
+- A button for updating general information,
+- A button for deleting the property from the list,
+- And a "Block periods" button for creating blocked periods.
+- If the "Block periods" button is clicked, the page shows a form with two input fields (or a calendar) to enter start and end dates for the block period, and a button to save the new blocked period. After saving the blocked period, the page shows again the list of uploaded properties for this user, which contains a button to navigate to the homepage.
+
+E) In the navbar I can see a button to show the list of all the reservations created by the logged-in user, with the name of the property, reservation Id, and a button to delete it.
+
+
+## Entity Relation Diagram:
+
+<img src="/images/ERD.jpeg" alt="Logo" width="800" height="500">
+  </a>
+
+## Built With:
+This project was built using these technologies:
+* Ruby
+* Ruby on Rails
+* React
+* Redux
+* HTML
+* CSS
+* JavaScript
+* PostgreSQL, pgAdmin 4
+* JSON Web Token (JWT)
+* API integration
+* Adobe Illustrator
+* Adobe After Effect
+* Rubocop
+* Capybara
+* Rspec
+* OpenApi(Swagger)
+* GitHub, Gitflow
+* Visual Studio
+
+
+## Usage:
+
+To have this app on your pc, you need to:
+
+  - Clone with SSH:
+  ```
+git@github.com:Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+  - Clone with HTTPS
+  ```
+https://github.com/Home-Sharing-Team/final-capstone-Project-Back-End.git
+  ```
+
+* In the project directory, you can run:
+
+  - `$ bundle install` - installs all the dependencies required by the project
+
+  Run 'Rails s' command to run the Ruby on Rails server and launch your browser to see the project
+
+## Testing:
+
+Tu run the test in your terminal, please type: `rspec spec/capybara`
+
+
```

Comment:
- [ ] When I run the tests according to your instructions, I'm greeted with this error:

![image](https://user-images.githubusercontent.com/57421823/203664768-0d6a84eb-e9e4-4ee0-b914-c616d26c4499.png)

Please, check if the tests are working on your end.  If your test setup requires special instructions to be run correctly, please add it to your readme.  If this is a bug, kindly fix it to ensure your app's integrity.

That way, your users will be able to run your tests smoothly 👍